pernstrong/refactor-tractor
# Feature/sass

#### What's this PR do?
This PR is our group pair/collaboration efforts for converting regular CSS into SASS. We also changed two of the cards colors, and corresponding icons, in order to make it more accessible for those with colorblindness. 
#### Where should the reviewer start?
scss files...base.scss, _variables.scss, _mixins.scss.
#### How should this be manually tested?
Click through the application to verify there is no off colors or formatting for SASS. Also can use the Dalton colorblind extension and make sure the color scheme works.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
